### PR TITLE
Connect MySQL and Change id's type to Integer

### PR DESCRIPTION
**Pull request by sim4858**
Connect MySQL and Change id's type to Integer

### CODE REVIEW

File: src/main/java/us/controller/MeetingController.java [hunk 1]
```diff
@@ -29,6 +29,7 @@ public String getCreateMeetingForm(Model model) {
 
     @RequestMapping(value = "/new", method = RequestMethod.POST)
     public String createMeeting(Meeting meeting) {
+        System.out.println(meeting.getName());
```
Comment (NESOY) [inline src/main/java/us/controller/MeetingController.java]: Please remove Test Code

**Comment by NESOY** [comment]
Checked this pull-request

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+spring.jpa.hibernate.ddl-auto=update
+
+spring.jpa.show-sql=true
+
+
+spring.datasource.url=jdbc:mysql://localhost:3306/nelp
+spring.datasource.username=root
+spring.datasource.password=csedbadmin
+spring.datasource.driverClassName=com.mysql.jdbc.Driver
```
Comment (cjh5414) [inline src/main/resources/application.properties:9]: Set DB user Info to Environmnet variable plz

Comment (cjh5414) [reply src/main/resources/application.properties:9]: https://softwareengineering.stackexchange.com/questions/246412/database-password-problem-when-using-git-to-manage-project-that-uses-spring-to-i/246413

**Comment by cjh5414** [comment]
Good

**Pull request merged**